docs/Reference/RPC/commandsRPC.md

```diff
@@ -245,7 +245,7 @@ rpc maintenance changepassword -u wss://server/maintenance
 
 <br>
 
-#### syncclock
+#### syncclock (maintenance)
```

Comment:
All these comes under configure not maintenance

Reply:
See line 602 for the docs for `configure` command version of syncclock. 

As far as I'm aware we still maintain the `maintenance` commands until we officially deprecate them. Because of that, I had to specify in the header between (maintenance) and (configure)